server/controllers/admin/allRequests.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable camelcase */
+import { Request, Response, NextFunction } from 'express';
+
+import { requestQuery } from '../../queries';
+import { CustomError } from '../../utils';
+
+const allRequests = async (req : Request, res : Response, next : NextFunction) => {
+  const { numOffSet } = req.body;
+  try {
+    const data = await requestQuery(numOffSet);
+    if (data.rows.length !== 0) {
+      res.json({ data });
+    } else {
+      throw new CustomError(400, 'There are no data');
+    }
```

Comment:
Just send the data and if there is no data we will handle it in the front, it is not a bad request to not have data